admin/admin-classic-editor.php

```diff
@@ -171,23 +171,18 @@ public function post_lang_choice() {
 			wp_die( 0 );
 		}
 
-		$post_type_object = get_post_type_object( $post_type );
-
-		if ( empty( $post_type_object ) ) {
-			wp_die( 0 );
-		}
-
-		if ( ! current_user_can( $post_type_object->cap->edit_post, $post_ID ) ) {
-			wp_die( -1 );
+		$return = $this->model->post->update_language( $post_ID, $lang, $post_type );
+		if ( is_wp_error( $return ) ) {
+			// phpcs:disable PSR2.ControlStructures.SwitchDeclaration.TerminatingComment
```

Comment:
> Should it be better to use a break; statement in the first case even if we use wp_die() ?
> It makes possible not to use a PHPCS ignore rule.

Revert because of a PHPStan analysis detect that break statement is unreachable due to wp_die() call.